### PR TITLE
serve static files using django-pipeline

### PR DESCRIPTION
**Pull request by fcurella**

### CODE REVIEW

File: pydotorg/settings.py [hunk 1]
```diff
@@ -105,3 +107,5 @@
         },
     }
 }
+
+from .pipeline import *
```
Comment (fcurella) [inline pydotorg/settings.py]: not sure about this. I don't like huge settings files, but i'm not crazy about like star-imports either.

File: templates/base.html [hunk 2]
```diff
@@ -18,12 +18,17 @@
 
     <script src="{{ STATIC_URL }}js/libs/modernizr.js"></script>
 
+    {% compressed_css 'style' %}
+    {% compressed_css 'mq' %}
+    {% comment %}
```
Comment (fcurella) [inline templates/base.html:24]: i kept as a mnemonic device, so i can remember the output of the `{% compressed_*`` tags

File: pydotorg/pipeline.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            'js/script.js',
+        ),
+        'output_filename': 'js/main.js',
+    }
+}
+
+PIPELINE_COMPILERS = (
+  'pipeline.compilers.sass.SASSCompiler',
+)
+PIPELINE_SASS_BINARY = 'cd %s && exec /usr/bin/env sass'  % os.path.join(BASE, 'static')
```
Comment (fcurella) [inline pydotorg/pipeline.py:61]: `cd` here is necessary to import compass' `config.rb`

File: README.md [hunk 4]
```diff
@@ -34,6 +34,11 @@ This expects a local database named "python.org". If you need to change it:
 
     $ export DATABASE_URL=postgres://user:pass@host:port/dbname
 
+To compile anc compress static media, you will need compass and yui-compressor:
```
Comment (jacobian) [inline README.md]: Typo: "and"

File: pydotorg/settings.py (hunk 1 above)
Comment (jacobian) [inline pydotorg/settings.py]: I think it's OK, but if it wigs you out you could do `from .pipline import PIPELINE_JS, PIPELINE_CSS, ...`

File: templates/base.html (hunk 2 above)
Comment (jacobian) [inline templates/base.html:24]: Maybe add a note to that effect too?

**Comment by jacobian** [comment]
Couple of minor things I noticed, but nothing big. 

However, Travis is choking on this because of the missing built files. Is there a way to fake making pipeline work under tests so we can run tests and CI without building the static components? If not, we can update Travis to install that stuff, but it might be nicer for people helping out if getting compass working was optional-ish,

**Comment by jhogue** [comment]
Hello all,

Just did the wrong thing, and I pushed a branch I was working on instead of
publishing and issuing a pull request. I apologize.

the changes were front-end design-based with some additional JS and removal
of old JS scripts. Please let me know if this caused any issues, and, sorry
about that again.

j

On Tue, Feb 19, 2013 at 4:43 PM, Jacob Kaplan-Moss <notifications@github.com

> wrote:
> 
> Couple of minor things I noticed, but nothing big.
> 
> However, Travis is choking on this because of the missing built files. Is
> there a way to fake making pipeline work under tests so we can run tests
> and CI without building the static components? If not, we can update Travis
> to install that stuff, but it might be nicer for people helping out if
> getting compass working was optional-ish,
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/proevo/pythondotorg/pull/1#issuecomment-13801586.

## 

_J. Hogue_: Design Director at Project Evolution
t: 888.456.0467 | c: 401.831.1963

**Comment by fcurella** [comment]
Hi @jhogue 

I'm merging master into this branch and fixing the conflicts. No worries.

**Comment by jhogue** [comment]
Thanks. Hah, I might be too front-end to be adding code now that you guys
are in there. I'll make sure to follow the correct procedure next time.

On Tue, Feb 19, 2013 at 5:09 PM, Flavio Curella notifications@github.comwrote:

> Hi @jhogue https://github.com/jhogue
> 
> I'm merging master into this branch and fixing the conflicts. No worries.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/proevo/pythondotorg/pull/1#issuecomment-13803004.

## 

_J. Hogue_: Design Director at Project Evolution
t: 888.456.0467 | c: 401.831.1963

**Comment by jacobian** [comment]
LGTM, merge when you're ready.

**Pull request merged**